### PR TITLE
fix(magicString): EVAL-343 change string for magic string in event bus and transaction

### PR DESCRIPTION
**Pull request by simonchaval**
## Describe your changes
Adding and changing existants magic strings to new magic strings.
For now only events bus and transations were modified.
## Issue ticket number and link
EVAL-327 : https://entsupport.gdapublic.fr/browse/EVAL-327
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/competences/service/impl/DefaultExportService.java [hunk 1]
```diff
@@ -1670,16 +1671,16 @@ public void getElevesExportReleveComp(final String finalIdClasse, String idStruc
                     handlerToAsyncHandler(message -> {
                         JsonObject body = message.body();
 
-                        if (OK.equals(body.getString(STATUS))) {
-                            JsonArray results = body.getJsonArray(RESULTS);
+                        if (Field.OK.equals(body.getString(STATUS))) {
+                            JsonArray results = body.getJsonArray(Field.RESULTS);
```
Comment (lapeyronnie) [inline src/main/java/fr/openent/competences/service/impl/DefaultExportService.java:1675]: tu as oublié STATUS

File: src/main/java/fr/openent/competences/service/impl/DefaultExportService.java [hunk 2]
```diff
@@ -1702,22 +1703,22 @@ public void getElevesExportReleveComp(final String finalIdClasse, String idStruc
...
                                     .put(Competences.ID_ETABLISSEMENT_KEY, idStructure)
                                     .put("idEleves", new JsonArray(Arrays.asList(idEleves)));
                             eb.send(Competences.VIESCO_BUS_ADDRESS, infosAction, Competences.DELIVERY_OPTIONS,
                                     handlerToAsyncHandler(event -> {
                                         JsonObject body = event.body();
 
-                                        if (OK.equals(body.getString(STATUS))) {
-                                            JsonArray results = body.getJsonArray(RESULTS);
+                                        if (Field.OK.equals(body.getString(STATUS))) {
+                                            JsonArray results = body.getJsonArray(Field.RESULTS);
```
Comment (lapeyronnie) [inline src/main/java/fr/openent/competences/service/impl/DefaultExportService.java:1714]: STATUS

File: src/main/java/fr/openent/competences/service/impl/DefaultMatiereService.java [hunk 3]
```diff
@@ -357,15 +358,15 @@ public void getSousMatieres(String idMatiere, String idStructure, Handler<Either
...
                 .put("userType", "Personnel")
                 .put("idUser", "null")
                 .put("idStructure", idEtablissement)
                 .put("onlyId", false);
         eb.send(Competences.VIESCO_BUS_ADDRESS, action, DELIVERY_OPTIONS, handlerToAsyncHandler(message -> {
             JsonObject body = message.body();
-            if (OK.equals(body.getString(STATUS))) {
-                handler.handle(new Either.Right<>(body.getJsonArray(RESULTS)));
+            if (Field.OK.equals(body.getString(STATUS))) {
+                handler.handle(new Either.Right<>(body.getJsonArray(Field.RESULTS)));
```
Comment (lapeyronnie) [inline src/main/java/fr/openent/competences/service/impl/DefaultMatiereService.java:369]: STATUS

File: src/main/java/fr/openent/competences/service/impl/DefaultExportService.java [hunk 4]
```diff
@@ -1645,13 +1646,13 @@ else if (idCycle == 2)
         }
         eb.send(Competences.VIESCO_BUS_ADDRESS, action, Competences.DELIVERY_OPTIONS, handlerToAsyncHandler(message -> {
             final JsonObject body = message.body();
-            if (OK.equals(body.getString(STATUS))) {
-                String libellePeriode = body.getString(RESULT)
+            if (Field.OK.equals(body.getString(STATUS))) {
+                String libellePeriode = body.getString(Field.RESULT)
```
Comment (lapeyronnie) [inline src/main/java/fr/openent/competences/service/impl/DefaultExportService.java:1650]: oublie STATUS

**Comment by lapeyronnie** [comment]
Je n'ai pas vu la classe java Competences sans les constantes que tu as déplacé dans la classe field.

**Comment by simonchaval** [comment]
Wrong method, now gonna use file the existant "Field.java" with magic string key to update in the continuity of the project.
cf : [This PR](https://github.com/OPEN-ENT-NG/competences/pull/127)

**Pull request closed without merging**